BrowserSelect/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="ini-parser" version="2.5.2" targetFramework="net40" />
```

Comment:
Please also include changes to the `.csproj` file, specifically this addition:
```xml
<Reference Include="INIFileParser, Version=2.5.2.0, Culture=neutral, PublicKeyToken=79af7b307b65cf3c, processorArchitecture=MSIL">
      <HintPath>..\packages\ini-parser.2.5.2\lib\net20\INIFileParser.dll</HintPath>
    </Reference>
```
Otherwise it won't compile
![image](https://user-images.githubusercontent.com/4184939/98870880-326c2600-2489-11eb-8470-8f6794272ff5.png)